app/helpers/job_invocation_output_helper.rb

```diff
@@ -17,13 +18,13 @@ module JobInvocationOutputHelper
   }.tap { |h| h.default = 'default' }.freeze
 
   def colorize_line(line)
-    line = line.gsub(/\e\[.*?m/) do |seq|
+    line = line.gsub(COLOR_PATTERN) do |seq|
       color = seq[/(\d+)m/,1]
       "{{{format color:#{color}}}}"
     end
 
     current_color = 'default'
-    out = %{<span style="color: #{@current_color}">}
+    out = %{<span style="color: #{current_color}">}
```

Comment:
Isn't the issue now, that the colorization doesn't survive the the new line?
[rhlogo.zip](https://github.com/theforeman/foreman_remote_execution/files/2396250/rhlogo.zip)

Reply:
![rhlogo](https://user-images.githubusercontent.com/190443/45742778-029f9b80-bbfb-11e8-93a8-373f156123ef.png)
The whole hat should be red